tests/queries/0_stateless/03002_part_log_rmt_fetch_merge_error.sh

```diff
@@ -33,13 +33,14 @@ $CLICKHOUSE_CLIENT -m -q "
     optimize table rmt_master final settings alter_sync=1, optimize_throw_if_noop=1;
 "
 
+# wait until rmt_slave will try to fetch the part and reflect this error in system.part_log
+wait_until "select count()>0 from system.part_log where table = 'rmt_slave' and database = '$CLICKHOUSE_DATABASE' and error > 0"
```

Comment:
Maybe we should execute `system flush logs part_log` each try? Otherwise the duration will likely be increased to 7.5 seconds likely

Reply:
Sounds reasonable. I've also increased the sleep time from 0.5 to 1 (seems better in case of flushing logs each iteration).